azure/cosmos/__init__.py

```diff
@@ -33,6 +37,45 @@ class User:
     pass
 
 
+class PartitionKey(dict):
+    """ Key used to partition a container into logical partitions.
+
+    See https://docs.microsoft.com/en-us/azure/cosmos-db/partitioning-overview#choose-partitionkey for more information
```

Comment:
Minor: URLs should not include locale identifiers (`en-us`) so browsers can be redirected automatically to the localized versions of articles.

Reply:
Updated (removed local identifier)